test/create_db.sh

```diff
@@ -22,33 +53,47 @@ mysql $dbconn -e "SET GLOBAL max_connections = 500;"
 for dbenv in $DBENVS; do
   db="boulder_sa_${dbenv}"
 
+  print_heading "Checking if "$db" exists"
   if mysql $dbconn -e 'show databases;' | grep $db > /dev/null; then
-    echo "Database $db already exists - skipping create"
+    echo ""$db" already exists - skipping create"
   else
-    create_script="drop database if exists \`${db}\`; create database if not exists \`${db}\`;"
-
-    mysql $dbconn -e "$create_script" || die "unable to create ${db}"
+    echo ""$db" doesn't exist - creating"
+    create_empty_db
+  fi
 
-    echo "created empty ${db} database"
+  # determine which db_path to use
+  if [[ "$BOULDER_CONFIG_DIR" = "test/config-next" ]]
+  then
+    dbpath="./sa/_db-next/"
+    migrations="./sa/_db-next/migrations"
+  else
+    dbpath="./sa/_db/"
+    migrationspath="./sa/_db/migrations"
```

Comment:
These are inconsistent; you use `migrations` in one place and `migrationspath` in another. I think it would be better to only set `dbpath` here, and when needed, do `${dbpath}/migrations`.

Reply:
yeah, this was just a mistake 👍🏻